src/main/java/com/chrosciu/bootcamp/tasks/input/InputUtils.java

```diff
@@ -5,11 +5,15 @@
 import reactor.core.publisher.Flux;
 
 import java.io.InputStream;
+import java.util.Scanner;
 
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class InputUtils {
     public static Flux<String> toFlux(InputStream inputStream) {
-        //TODO: Implement
-        return null;
+        Scanner sc = new Scanner(inputStream);
+        return Flux.create(sink -> {
+            sink.next(sc.nextLine());
```

Comment:
to stworzy strumień który zakończy się po wczytaniu pierwszej linii :) a ma działać tak, że będzie wczytywać linie w nieskończoność